components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -142,7 +142,7 @@ paths:
             - apim:admin
             - apim:tier_view
       x-code-samples:
-        - lang: Shell
+        - lang: Curl
```

Comment:
@vithu30  Do we have a redocly command as curl? is this correct?

Reply:
Yes. It's curl. Had an offline discussion with Dilmi and it's verified locally. +1 to this change.